### ISSUE TITLE
[Bug] [zeta] rest api submit a job with chinese name,seatunnel.sh -l returns an Garbled code name

### ISSUE THREAD

**Issue by gitfortian** [apache/seatunnel#5860]
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/seatunnel/issues?q=is%3Aissue+label%3A%22bug%22) and found no similar issues.

### What happened

seatunnel.sh - l
returns

776268698427064321  %E4%BA%BA%E5%91%98%E4%BF%A1%E6%81%AF%E5%AE%9E%E6%97%B6%E5%90%8C%E6%AD%A5  FINISHED    2023-11-13 10:14:40.778  2023-11-15 13:56:06.5

jobname %E4%BA%BA%E5%91%98%E4%BF%A1%E6%81%AF%E5%AE%9E%E6%97%B6%E5%90%8C%E6%AD%A5

### SeaTunnel Version

2.3.3

### SeaTunnel Config

```conf
{"env":{"execution.parallelism":"2","job.mode":"BATCH","checkpoint.interval":"10000"},"source":[{"url":"jdbc:mysql://127.0.0.1:3306/erp?useSSL=false&characterEncoding=UTF-8","user":"root","password":"","driver":"com.mysql.cj.jdbc.Driver","connection_check_timeout_sec":100,"query":"select * from employee where 1=1","plugin_name":"Jdbc","result_table_name":"fake"}],"sink":[{"plugin_name":"Doris","fenodes":"","username":"root","password":"123456","sink.label-prefix":"ods_ods_erp_employee_1700019993157","table.identifier":"ods.ods_erp_employee","doris.config":{"format":"json","read_json_by_line":"true"}}]}
```

### Running Command

```shell
String url = "http://" + node + "/hazelcast/rest/maps/submit-job?jobName=" + taskName + "&jobId=" + jobId;
jobname='这个是中文任务名称测试任务'
```

### Error Exception

```log
jobname %E4%BA%BA%E5%91%98%E4%BF%A1%E6%81%AF%E5%AE%9E%E6%97%B6%E5%90%8C%E6%AD%A5
```

### Are you willing to submit PR?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

**Comment by zhibinF** [comment]
I am willing to fix it.

**Comment by liugddx** [comment]
#5870

**Comment by liugddx** [comment]
> #5870

This bug has been fixed, please check #5870
